### ISSUE TITLE
A11Y: Logo is not labeled properly.

### ISSUE THREAD

**Issue by mmamu** [ampproject/amphtml#4980]
Environment:
Device: Ipod Touch

Pre- Requisite: Turn Voiceover On.

Steps to Reproduce:
1. Open (https://amp-mk-3.herokuapp.com/examples/viewer.html) In incognito mode. 
2. Click on "Two" in the menu section.
3. Linear navigate until you reached to the banner at the bottom of the page or touch on the banner.

Expected Result:
Logo label should provide meaningful information to the user.

Actual Result:
Logo is read as "onestart...PNG Image".

Please see the attachment for details.
![img_0002](https://cloud.githubusercontent.com/assets/22181621/18492219/ebeea532-79be-11e6-9a0b-93596250284a.jpg)

**Comment by ericlindley-g** [comment]
Thanks for checking the a11y for this feature, @mmamu — this logo labeling is something that publishers will have control of, and isn't expected to work in the demo, so I'm closing this as WAI.
